src/components/LabelledSelectInput.tsx

```diff
@@ -12,9 +12,14 @@ export interface InputProps {
 	children?: string | React.ReactNode;
 	value?: any;
 	width?: string;
+	disabled?: boolean;
 }
 
-const LabelledSelectInput = ({ label, placeholder, set, visible, value, width }: InputProps) => {
+const LabelledSelectInput = ({ label, placeholder, set, visible, value, width, disabled }: InputProps) => {
+	useEffect(() => {
+		console.log(disabled);
```

Comment:
remove this effect in the next refactor

Reply:
this effect is added to make dropdown disabled for a summary page

Reply:
but its only a console.log right ? the disable works without the useEffect right ?